ReactiveCocoaTests/Swift/TestError.swift

```diff
@@ -14,3 +17,27 @@ enum TestError: Int {
 
 extension TestError: ErrorType {
 }
+
+
+internal extension SignalProducerType {
+	/// Halts if an error is emitted in the receiver signal.
+	/// This is useful in tests to be able to just use `startWithNext`
+	/// in cases where we know that an error won't be emitted.
+	func assumeNoErrors() -> SignalProducer<Value, NoError> {
+		return self.lift { $0.assumeNoErrors() }
+	}
+}
+
+internal extension SignalType {
+	/// Halts if an error is emitted in the receiver signal.
+	/// This is useful in tests to be able to just use `startWithNext`
+	/// in cases where we know that an error won't be emitted.
+	func assumeNoErrors() -> Signal<Value, NoError> {
```

Comment:
Yeah, I do like how this version neatly captures the assumption that there will be no errors.

I do wonder if there is a form that makes it _slightly_ less easy to introduce crashes—but on the other hand if you're going to misuse the API there's not much we can do to prevent it, other than not exposing the operator publicly.